src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -196,6 +196,12 @@ namespace
             user->access.revokeGrantOption(AccessType::ALL);
         }
 
+        String default_database = config.getString(user_config + ".default_database", "");
+        if (!default_database.empty())
```

Comment:
`if` is not necessary, nothing bad will happen if you assign `user->default_database` to an empty string here.